metricflow/plan_conversion/select_column_gen.py

```diff
@@ -33,11 +33,11 @@ def as_tuple(self) -> Tuple[SqlSelectColumn, ...]:
         """Return all select columns as a tuple."""
         return tuple(
             # Preferred order is to have metrics first.
-            self.metric_columns
+            self.identifier_columns
```

Comment:
That comment is now LIES! LIES I tell you!

Uh, sorry, maybe the iced coffee was a mistake. Please remove that comment.